sklearn/tests/test_common.py

```diff
@@ -332,7 +332,9 @@ def _generate_search_cv_instances():
         extra_params = (
             {"min_resources": "smallest"} if "min_resources" in init_params else {}
         )
-        search_cv = SearchCV(Estimator(), param_grid, cv=2, **extra_params)
+        search_cv = SearchCV(
+            Estimator(), param_grid, cv=2, error_score="raise", **extra_params
```

Comment:
Setting `error_score="raise"` makes pytest traceback reading much more direct, especially in CI logs.